### ISSUE TITLE
Recomendations for Hardware?

### ISSUE THREAD

**Issue by ph1l** [rob42/freeboard-server#19]
I'm building out a freeboard-server nav system for my boat. I have no working electronics currently, so I've acquired a GPS and and the peet brothers anemometer in addition to the Raspberry Pi, Arduino and Freeboard Interface Board V1.2.

In my testing, the GPS Unit and Wind instrument seem to work great out of the box. I'm currently looking to get a depth sounder and paddle wheel speed sensor. Any recommendations on how to get compatible hardware and hook it up?

**Comment by rob42** [comment]
Hi Phil,

Get a depth sounder that can output NMEA0183, and connect it via the freeboard interface board NMEA ports. There is support for a cool depth sounder with a sparkline and alarm depth in latest Freeboard courtesy of Ron Berliner :-)

Rob

June 8 2016 3:28 PM, "Philip Freeman"  wrote:
	I'm building out a freeboard-server nav system for my boat. I have no working electronics currently, so I've acquired a GPS and and the peet brothers anemometer in addition to the Raspberry Pi, Arduino and Freeboard Interface Board V1.2.

	In my testing, the GPS Unit and Wind instrument seem to work great out of the box. I'm currently looking to get a depth sounder and paddle wheel speed sensor. Any recommendations on how to get compatible hardware and hook it up?

**Comment by ph1l** [comment]
Is there anything special about logs?

**Comment by rob42** [comment]
Basically any suitable paddlewheel can be connected to the log connectors. They work by hall effect or need switch , so they do vary. 2 Wires is a reed switch, More is hallsensor.
Check they output <5v. Needs some code as I just use the GPS

**Comment by RBerliner** [comment]
Take a look at the Actrisense instruments. They have a device that powers
and interprets a 200 kHz depth sounder and also accepts paddlewheel log
signals. It puts out NMEA which is compatible with freeboard.

On Wed, Jun 8, 2016 at 1:05 AM, rob42 <notifications@github.com> wrote:

> Hi Phil,
>
> Get a depth sounder that can output NMEA0183, and connect it via the
> freeboard interface board NMEA ports. There is support for a cool depth
> sounder with a sparkline and alarm depth in latest Freeboard courtesy of
> Ron Berliner :-)
>
> Rob
>
> June 8 2016 3:28 PM, "Philip Freeman" wrote:
> I'm building out a freeboard-server nav system for my boat. I have no
> working electronics currently, so I've acquired a GPS and and the peet
> brothers anemometer in addition to the Raspberry Pi, Arduino and Freeboard
> Interface Board V1.2.
>
> In my testing, the GPS Unit and Wind instrument seem to work great out of
> the box. I'm currently looking to get a depth sounder and paddle wheel
> speed sensor. Any recommendations on how to get compatible hardware and
> hook it up?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub (
> https://github.com/rob42/freeboard-server/issues/19), or mute the thread (
> https://github.com/notifications/unsubscribe/ADFtXdimYSDiInRLvWtLWUjtkNdoXlzjks5qJjbWgaJpZM4Iwj96
> ).
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rob42/freeboard-server/issues/19#issuecomment-224488429>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADgpSctCUrTWpcG01EdR4ngdfAPLm_Opks5qJk2rgaJpZM4Iwj96>
> .
>



-- 

R. Berliner
Instrumentation Associates
101 Echowood Ln
Cary, NC 27518

e-mail: rberliner@InstrumentationAssociates.com

www.InstrumentationAssociates.com <http://www.InstrumntationAssociates.com>

Tel     734-646-5318

**Comment by rob42** [comment]
This shows the basic operation https://www.bakesonline.com/images/MediaLibrary/PaddleWheelTest.jpg
To use this on the freeboard interface you could try powering with 5V, or you need to use a resistor to limit voltage to 5V on the green wire.

**Comment by rob42** [comment]
And for airmar http://www.catalina320.com/article.php?story=speedotesting

**Comment by rob42** [comment]
The freeboardPLC code for the arduino doesnt currently have a log function, but adding it should be pretty simple. Its just a count of pulses/time to get a basic reading. Multiply that by a factor to get the calibration right.

**Comment by ph1l** [comment]
I have a NMEA0183 Depth Sounder, but not seeing any data coming through in the logs when I hook it up. I have connected the device to +12V and ground, and put the NMEA Out fromthe device to NMEA1's RX connection on the board. Am I missing something?

thanks.

**Comment by rob42** [comment]
Check the baud rate that the depth sounder uses. It must match the NMEA1 baud, which is 9600bps. If the depth sounder cant be changed to 9600 you can change the freeboard interface by connecting with a terminal and sending "#SB2=4800" (without the ""). I usually use the arduino ide serial console.

Rob

September 25 2016 6:57 AM, "Philip Freeman"  wrote:
	I have a NMEA0183 Depth Sounder, but not seeing any data coming through in the logs when I hook it up. I have connected the device to +12V and ground, and put the NMEA Out fromthe device to NMEA1's RX connection on the board. Am I missing something?

	thanks.

**Comment by ph1l** [comment]
Yup it was totally the baud rate, since the device has nothing but 3 wires coming out of it, I don't think there's a way to change it, but setting the port's rate to 4800 on the Arduino did the trick! Thanks
